MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -10,11 +10,11 @@
             <img class="login-phone-mode-logo" src="../../Content/Images/Logos/meetu-logo-160x160.png" />
             <img class="login-phone-mode-text" src="../../Content/Images/Logos/MeetU-text-240x80.png" />
         </div>
-        <div class="login-phone-mode-homepage-arrow-wrapper">
+        <div class="login-phone-mode-homepage-arrow-wrapper" id="jump_icon">
```

Comment:
since id is unique, we usually use class to do things like this. also, if some classes is used in js, the naming convention would be with js- prefix. 
eg. in this case, it is in login module, so we can set the name with prefix js-login-, then use the '-' to concatenate words. it will end up with 'js-login-jump-icon' and 'js-login-form'. naming convention is very important, which will save you a bunch of time.